### ISSUE TITLE
Signal service is down

### ISSUE THREAD

**Issue by seanlano** [signalapp/Signal-Android#7869]
- [x] I have searched open and closed issues for duplicates
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The Signal app on my Android 8.1 phone cannot connect or send any messages. The debug logs show: `WebSocketConnection: java.net.ConnectException: Failed to connect to textsecure-service.whispersystems.org/127.0.0.1:443`. 
It seems the DNS lookup is redirecting to `localhost`, presumably because of Denial-of-Service?

### Steps to reproduce
- Use the Signal app to send a message
- Get a "Not delivered" message

### Device info
**Device:** OnePlus 3T
**Android version:** 8.1.0
**Signal version:** 4.19.3

### Link to debug log
https://debuglogs.org/d64065db8bbc0f270edd3d0ce0d71b083935dcec358b9787db487675532cba83

**Comment by ggoss** [comment]
Same issue here, also on a 3T.

**Comment by seanlano** [comment]
Even if I ping the address from my desktop I get the `localhost` address:

```$ ping textsecure-service.whispersystems.org
PING textsecure-service.whispersystems.org (127.0.0.1) 56(84) bytes of data.
64 bytes from localhost (127.0.0.1): icmp_seq=1 ttl=64 time=0.052 ms
64 bytes from localhost (127.0.0.1): icmp_seq=2 ttl=64 time=0.037 ms
```

**Comment by Dyras** [comment]
Sweden here. It's down here as well!

Interestingly, the latest beta update was downloaded while it was down though 😕

**Comment by Vaelatern** [comment]
Having issues on both Desktop (Linux Electron) and Android.

**Comment by Bit4BitForensics** [comment]
Both Android and Windows Desktop having issues after update as well here.

**Comment by mattookay** [comment]
Android Samsung Galaxy S7 wont send or recieve any messages.

**Comment by keeepinitgansta** [comment]
Yep, Signal is down because the DNS records are pointing to local host (127.0.0.1).

Here is the link to the discussion over on the Signal-Desktop forum

https://github.com/signalapp/Signal-Desktop/issues/2428

**Comment by lambda** [comment]
I'm getting an alternation between resolving to `127.0.0.1`, and the following results:

```
textsecure-service.whispersystems.org. 56 IN A	52.201.16.192
textsecure-service.whispersystems.org. 56 IN A	52.4.181.2
textsecure-service.whispersystems.org. 56 IN A	52.202.234.33
textsecure-service.whispersystems.org. 56 IN A	52.54.209.68
textsecure-service.whispersystems.org. 56 IN A	35.170.152.207
textsecure-service.whispersystems.org. 56 IN A	52.23.49.187
textsecure-service.whispersystems.org. 56 IN A	52.70.150.128
textsecure-service.whispersystems.org. 56 IN A	52.45.155.28
```

**Comment by lambda** [comment]
If I add those above results to my router's `/etc/hosts`, I am still getting an unexpected end of stream error.

**Comment by rawells14** [comment]
Both android and native linux app are down.

**Comment by mapinis** [comment]
It seems to be back up, I'm both receiving and sending from Android and Windows Desktop.

**Comment by keeepinitgansta** [comment]
Yep it's back and signal posted on twitter in regards

https://twitter.com/signalapp/status/1002336532903022592?s=09
